crates/sui/tests/snapshots/ptb_files_tests__transfer_objects_valid.ptb.snap

```diff
@@ -1,30 +1,6 @@
 ---
 source: crates/sui/tests/ptb_files_tests.rs
-expression: "results.join(\"\\n\")"
+expression: "format!(\"=== FILE EXPANSION ERROR === \\n {:?}\", e)"
 ---
- === PREVIEW === 
-╭───────────────────────────────────────────────────────────────────────────────────────────────────────╮
-│ PTB Preview                                                                                           │
-├──────────────────┬─────────────────────────────────────────────────────────────┬──────────────────────┤
-│ command          │ from                                                        │ value(s)             │
-├──────────────────┼─────────────────────────────────────────────────────────────┼──────────────────────┤
-│ gas_budget       │ tests/ptb_files/transfer_objects/transfer_objects_valid.ptb │ 10000000             │
-│ split_coins      │ tests/ptb_files/transfer_objects/transfer_objects_valid.ptb │ gas [1,2,3]          │
-│ assign           │ tests/ptb_files/transfer_objects/transfer_objects_valid.ptb │ s                    │
-│ transfer_objects │ tests/ptb_files/transfer_objects/transfer_objects_valid.ptb │ @0x1 [s.0, s.1, s.2] │
-│ preview          │ console                                                     │ true                 │
-╰──────────────────┴─────────────────────────────────────────────────────────────┴──────────────────────╯
- === PARSED INPUT COMMANDS === 
-cmd: file-include-start, value: ["\"tests/ptb_files/transfer_objects/transfer_objects_valid.ptb\""]
-cmd: gas_budget, value: ["10000000u64"]
-cmd: split_coins, value: ["gas", "[1u64, 2u64, 3u64]"]
-cmd: assign, value: ["s"]
-cmd: transfer_objects, value: ["@0x1", "[s.0, s.1, s.2]"]
-cmd: preview, value: ["false"]
-cmd: warn_shadows, value: ["false"]
-cmd: file-include-end, value: ["\"tests/ptb_files/transfer_objects/transfer_objects_valid.ptb\""]
-cmd: preview, value: ["true"]
-cmd: warn_shadows, value: ["false"]
- === BUILT PTB === 
-SplitCoins(GasCoinInput(0),Input(1),Input(2))
-TransferObjects([NestedResult(0,0),NestedResult(0,1),NestedResult(0,2)],Input(3))
+=== FILE EXPANSION ERROR === 
```

Comment:
This is a cargo insta issue sadly :'( I just checked this out locally since I was puzzled by this exact same test.

If you run with `cargo nextest run` this file will be correct, but for some reason cargo insta doesn't work with it